docs/topics/development/setup_and_configuration.md

```diff
@@ -0,0 +1,316 @@
+# Setup and Configuration
+
+## Local Development Environment
+
+Setting up the local development environment for the **addons-server** involves configuring Docker Compose to run the necessary services. Follow these steps to get started:
+
+1. **Prerequisites**:
+    - Ensure Docker and Docker Compose are installed on your system.
+    - Clone the **addons-server** repository from GitHub:
+
+      ```sh
+      git clone https://github.com/mozilla/addons-server
+      cd addons-server
+      ```
+
+2. **Configuration**:
+    - Run the following command to create the `.env` file with necessary environment variables:
+
+      ```sh
+      make setup
+      ```
+
+      This command generates the `.env` file required by our Docker Compose configuration in the `env_file` property. The `.env` file includes sensible defaults for each required value. Changes to the `.env` file will persist if you rerun `make setup`.
+
+3. **Bringing Up Services**:
+    - When running the project for the first time, execute:
+
+      ```sh
+      make initialize_docker
+      ```
+
+      This command will:
+      - Run `make up` to start the Docker containers.
+      - Run `make initialize` to set up the initial Docker environment, including database initialization and data population. Detailed steps for `make initialize` will be covered in Section 6 on Data Management.
+
+4. **Understanding `make up`**:
```

Comment:
We should make it clearer to new users that `make up` is how you start `addons-server` each subsequent time

Reply:
Wdyt?